src/app/pages/swap/components/swap-details/swap-details.tsx

```diff
@@ -76,8 +79,8 @@ export function SwapDetails() {
       />
       <SwapDetailLayout
         title="Liquidity provider fee"
-        tooltipLabel="To receive a share of these fees, become a Liquidity Provider on app.alexlab.co."
-        value={`${swapSubmissionData.liquidityFee} ${swapSubmissionData.swapAssetBase.name}`}
+        tooltipLabel="To receive a share of these fees, become a Liquidity Provider on app.bitflow.finance."
```

Comment:
Should this be dynamic and change between `alexlab.co`  & `bitflow.finance`?

Reply:
Good question. Technically the underlying liquidity pools can span multiple DEXes. Might be weird to try and accommodate any / all LPs Bitflow could use in a swap. Maybe better to remove the comment? @markmhendrickson thoughts?

Reply:
I think we can remove this tooltip in general and revisit later when we redesign swaps for mobile

Reply:
As Mark says we can remove, but to make generic we could add a `providerDomain` to the context, and read that so `<SwapDetails />` doesn't know about any given provider